feature/my/src/main/java/com/hankki/feature/my/component/EmptyStoreView.kt

```diff
@@ -26,9 +27,12 @@ import com.hankki.feature.my.R
 fun EmptyStoreView(
     text: String,
     modifier: Modifier = Modifier,
+    navigateToHome: (() -> Unit)? = null
```

Comment:
wow
null은 안됩니다!!!!!!!!!!
```suggestion
    navigateToHome: (() -> Unit) = {}
```

Reply:
크헉 사실 제시해주신 방법으로 했다가 조건문으로 비교하는 방법을 못찾아서 무심코.. null을 썼던 것 같습니다
좀 더 찾아보고 코드 수정하겠습니다..!